server/src/test/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java

```diff
@@ -360,28 +346,24 @@ public void testReplicaRecovery() throws Exception {
 
         // we should now have two total shards, one primary and one replica
         execute("SELECT * FROM sys.shards WHERE table_name = '" + INDEX_NAME + "'");
-        assertThat(response.rowCount(), is(2L));
+        assertThat(response).hasRowCount(2L);
 
         var indexName = IndexParts.toIndexName(sqlExecutor.getCurrentSchema(), INDEX_NAME, null);
         final RecoveryResponse response = client().execute(RecoveryAction.INSTANCE, new RecoveryRequest(indexName)).get();
 
         // we should now have two total shards, one primary and one replica
         List<RecoveryState> recoveryStates = response.shardRecoveryStates().get(indexName);
-        assertThat(recoveryStates.size(), equalTo(2));
+        assertThat(recoveryStates.size()).isEqualTo(2);
```

Comment:
very minor, no need to address: here and in other places, you can use directly `.hasSize()`